Sources/Timer.swift

```diff
@@ -17,9 +17,16 @@ struct Timer {
     init(startingAt startingTimeInMilliseconds: Double = 0.0) {
         var startTime = timeval()
         gettimeofday(&startTime, nil)
-        if !startingTimeInMilliseconds.isZero {
-            startTime.tv_usec += type(of: startTime.tv_usec).init(startingTimeInMilliseconds * 1000)
```

Comment:
`tv_usec` overflows on arch32 when passing in a value which is bigger than `2147483647` which means that the limit of startingTime is ~34 minutes